tools/config/Tomb1Main_ConfigTool/Resources/Lang/es.json

```diff
@@ -195,6 +195,10 @@
       "Description": "Permite que los sonidos del juego continúen sonando en la pantalla de inventario.",
       "Title": "Habilitar sonidos de juegos en el inventario"
     },
+    "load_current_music": {
+      "Title": "Load music track",
+      "Description": "TBD."
```

Comment:
Let's remember to change this

Reply:
@ajtudela would you mind translating again?

```
"Load music track",
"Loads the music track that was playing when the game was saved."
```

Reply:
````
"load_current_music": {
      "Title": "Cargar pista de sonido",
      "Description": "Carga la pista de música que se estaba reproduciendo cuando se guardó el juego".
}
````

Next time, if you don't mind, left the file "es.json" without changes. That way I can compare the file with the "en.json" using i18n Ally. I think it would be easier for boths.

Reply:
Ok. Thanks!